Stellar-operation-manage-asset.x

```diff
@@ -50,44 +51,68 @@ struct UpdateMaxIssuance {
     Result: ManageAssetResult
 */
 
+//: ManageAssetOp is used to:
+//: * create or update `CREATE_ASSET` request;
+//: * create or update `UPDATE_ASSET` request;
+//: * cancel `CREATE_ASSET` or `UPDATE_ASSET` request
+//: * change asset pre issuer
+//: * update max issuance of asset
 struct ManageAssetOp
 {
-	uint64 requestID; // 0 to create, non zero will try to update
+    //: Use zero to create request, non zero to manage existing request
```

Comment:
//: Use zero value to create a request and non zero value to manage an existing request